### DISCUSSION TITLE
Awesome extension, thank you!

### DISCUSSION THREAD

**Discussion by Ben136KBC** [microsoft/vscode-cpptools#7171] [General]
I just want to say I _**love**_ this extension. We use VS Code to open a folder contains mostly C++ and Fortran files, literally hundreds of thousands of files, and the intellisense and this extensions works like a dream. If it can't resolve something it explains. And it is nearly instant.

Visual Studio (NOT talking about VS Code here) can also open a folder, but their intellisense is problematic. It can be sluggish, often locks up or freezes for several seconds, sometimes can't find a symbol in the same file, and the behavior changes with every update. Version 16.9 totally broke it again, and I have several issues open with them. I have suggested they adopt whatever is in VS Code.

So THANK YOU for this awesome VS Code extension, THANK YOU. Please keep it just working as well as it is now! _Please_ do not break it.

I should mention I have:  "intelliSenseMode": "clang-x64",   in c_cpp_properties.json, on Windows. The code is compiled using Microsoft Visual C++ via custom tools.

**Comment by sean-mcmanus** [comment]
Thanks. We share several core components with VS such as the parsers, but the UI model is different, so blocking the UI thread is not possible like it is in VS. We occasionally take updates from VS (e.g. IntelliSense parser bug fixes), usually in our Insiders releases, so let us know if you hit any regressions from those.
